test/new_relic/agent/method_tracer_helpers_test.rb

```diff
@@ -0,0 +1,73 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+
+require File.expand_path(File.join(File.dirname(__FILE__), '..', '..', 'test_helper'))
+
+module ::The
+  class Example
+    def self.class_method; end
+    def instance_method; end
+  end
+end
+
+class NewRelic::Agent::MethodTracerHelpersTest < Minitest::Test
+  # TODO: trace_execution_scoped should have test coverage
```

Comment:
weird that there isn't any tests for it yet, although definitely not unheard of. I wonder if there are tests tucked into other files/suites that are testing?

Reply:
Prior to the CLM functionality being added to it, this module only did one thing - it defined a `trace_execution_scoped` method.

What's fascinating is that the only consumer of this module at the time was `MethodTracer`, which also defined its own `trace_execution_scoped` method. So the lone consumer of the method has a method with the same name (which presumably _does_ have test coverage) which defers everything over to the method in the helpers module. Perhaps there was an original intent to share the module more widely.